internal/service/alarms/internal/alarms_server.go

```diff
@@ -2,15 +2,17 @@ package internal
 
 import (
 	"context"
+	"fmt"
 	"net/http"
-	"time"
 
-	"github.com/google/uuid"
+	"github.com/openshift-kni/oran-o2ims/internal/service/alarms/internal/db/repo"
+
 	api "github.com/openshift-kni/oran-o2ims/internal/service/alarms/api/generated"
+	"github.com/openshift-kni/oran-o2ims/internal/service/alarms/internal/db/models"
 )
 
 type AlarmsServer struct {
-	AlarmsRepository *AlarmsRepository
+	AlarmsRepository *repo.AlarmsRepository
```

Comment:
you can mock this to have some basic unit tests

Reply:
Yeah absolutely but there's not much to test with this current PR (other than actually sql stuff) so not creating any interface for now to mock.  Will update this on a follow up PR where we have stuff outside of the sql calls to test...wdyt?